packages/cart/src/services/cart-module.ts

```diff
@@ -137,7 +137,33 @@ export default class CartModuleService implements ICartModuleService {
     data: CartTypes.CreateCartDTO[],
     @MedusaContext() sharedContext: Context = {}
   ) {
-    return await this.cartService_.create(data, sharedContext)
+    const lineItemsToCreate: CartTypes.CreateLineItemForCartDTO[] = []
+    const createdCarts: Cart[] = []
+    for (const { items, ...cart } of data) {
+      const [created] = await this.cartService_.create(
+        [cart as CartTypes.CreateCartDTO],
```

Comment:
suggestion: shouldn't the type for create cart here be an internal module type and not the public type?